e2e-tests/haproxy/run

```diff
@@ -19,8 +19,12 @@ check_haproxy_writer() {
 main() {
   create_infra $namespace
 
-  if [[ "$OPENSHIFT" != "3.11" ]]; then
-    apply_config "$test_dir/conf/docker-rc.yaml"
+  if [[ "${OPENSHIFT}" == 4 ]]; then
+    apply_config "$test_dir/conf/container-rc.yaml"
+  elif [[ -z "${OPENSHIFT}" ]] && [[ $(echo "$KUBE_VERSION >= 1.20" | bc -l) -eq 1 ]]; then
```

Comment:
In my environment I got some error because the `+` sign in version:
```
$ KUBE_VERSION=$(kubectl version -o json | jq -r '.serverVersion.major + "." + .serverVersion.minor')
$ echo $KUBE_VERSION
1.18+
$ echo "$KUBE_VERSION >= 1.20" | bc -l
(standard_in) 1: parse error
```

Reply:
@tplavcic thanks, fixed.